src/sink/gpub.rs

```diff
@@ -49,6 +51,7 @@ pub struct Config {}
 
 enum PubSubCommand {
     SendMessage(String, String, Value<'static>),
```

Comment:
named structs would be better here, it'd allow conveying information about what the different parts mean. something like:

```rust
    SendMessage{this_gets_a_descripteive_name: String, that_gets_a_descriptive_anem: String, data: Value<'static>},
``

some for the create command

Reply:
Oh nice ^_^ I didn't know we could do that too

Reply:
Ja named fields are super powerful, a bit more verbose to write but that pays off 3 weeks in the future when no one remembers which field meant what :D